ce/api/util.py

```diff
@@ -80,3 +81,19 @@ def mean_datetime(datetimes):
     timestamps = [ dt.replace(tzinfo=timezone.utc).timestamp() for dt in datetimes ]
     mean = np.mean(timestamps)
     return datetime.fromtimestamp(mean, tz=timezone.utc)
+
+def search_for_unique_ids(sesh, ensemble_name='ce', model='', emission='', variable='', time=0):
+    query = sesh.query(DataFile.unique_id)\
+                  .join(DataFileVariable, EnsembleDataFileVariables, Ensemble, Run, Model, Emission)\
+                  .filter(Ensemble.name == ensemble_name)\
+                  .filter(DataFileVariable.netcdf_variable_name == variable)\
+                  .filter(Time.time_idx == time)\
```

Comment:
If you're searching for seasonal averages (e.g. pseudo-timestep 14), but there exist files that only have monthly averages, (e.g. 0-11), those files should not be included in the search results.